syntax = "proto3";

package proj3;

service BankSystem {
  rpc MsgDelivery (MsgDeliveryRequest) returns (MsgDeliveryReply) {}
}

message MsgDeliveryRequest {
  string interface = 1;
  string money = 2;
  repeated int64 writeSet = 3;
}

message MsgDeliveryReply {
  string interface = 1;
  string result = 2;
  string money = 3;
}